src/screens/SeasonDetailsScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from 'react';
+import { getTvSeasonDetails } from '../helpers';
+import { SeasonDetails } from '../types';
+import { Location, useLocation, Link } from 'react-router';
+import { CardMedia, Typography as Typ } from '@mui/material';
```

Comment:
Regarding these imports, just be aware of https://github.com/Thenlie/Streamability/pull/1003. Basically make the imports as explicit as possible to reduce bundle size.